### DISCUSSION TITLE
Comments only on posts

### DISCUSSION THREAD

**Discussion by LITUATUI** [hbstack/theme-cards#231] [Q&A] [answered]
How to allow comments only on posts (exclude pages)?

**Comment by razonyang** [comment]
You can achieve this by setting `comments` as `false` on front matter for other pages.

```yaml
---
comments: false
---
````

**Reply by razonyang** [reply, marked as answer]
Similarly, you can use this with [cascade](https://gohugo.io/content-management/front-matter/#cascade) for bulk of pages.
